### ISSUE TITLE
Document: wrong examples of getting TLS certs and keys

### ISSUE THREAD

**Issue by dongsupark** [kyma-project/kyma#1968]
**Description**

[A document about installation on GKE](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/032-inst-gke-installation.md#get-the-tls-certificate) has wrong examples for getting TLS certs and keys:

```
export TLS_CERT=$(cat ./letsencrypt/live/$DOMAIN/fullchain.pem | base64 | sed 's/ /\\ /g')
export TLS_KEY=$(cat ./letsencrypt/live/$DOMAIN/privkey.pem | base64 | sed 's/ /\\ /g')
```

On Linux (Fedora 29 in my case) the commands above do not remove whitespaces in the base64 encoded strings. As a result, `$TLS_CERT` and `$TLS_KEY` end up having multiple whitespace characters. That's definitely not expected. Because of the wrong environment variables, the kyma-installer does not install anything. Helm install simply fails.

In case of Linux, the commands above should be like this:

```
export TLS_CERT=$(cat ./letsencrypt/live/$DOMAIN/fullchain.pem | base64 --wrap=0)
export TLS_KEY=$(cat ./letsencrypt/live/$DOMAIN/privkey.pem | base64 --wrap=0)
```

I'm not sure if that would also work on other OSes like Darwin.

**Comment by Demonsthere** [comment]
Hello,
We are aware of this problem, however this is a specific `base64` version problem. MacOS, Debian, Fedora have different version of the `base64` application, which work in a slightly different way. 
Currently, we can confirm that is working as expected on the default `base64` for MacOS, and Debian, and the Busybox based binary is faulty. 
Please verify with a different binary for `base64`

**Comment by tomekpapiernik** [comment]
This has been resolved by #3325. Closing.
